mergeboard/src/app/github-test/page.tsx

```diff
@@ -0,0 +1,31 @@
+"use client"
+
+import { useEffect, useState } from "react";
+
+
+interface Repo {
+  id: number;
+  name: string;
+}
+
+export default function GithubTest() {
+  const [repos, setRepos] = useState<Repo[]>([]);
+
+  useEffect(() => {
+    fetch("/api/github")
+    .then((res)=> res.json())
+    .then((data) =>setRepos(data));
+  }, []);
+
```

Comment:
import and call `getRepos` in this component. Something like -
```
const repos = await getRepos()
```
Remember to make the component async